internal/component/database_observability/postgres/collector/explain_plan.go

```diff
@@ -518,14 +519,17 @@ func (c *ExplainPlan) fetchExplainPlanJSON(ctx context.Context, qi queryInfo) ([
 		return nil, fmt.Errorf("failed to set plan cache mode: %w", err)
 	}
 
+	explainQuery := fmt.Sprintf("%s%s", selectExplainPlanPrefix, preparedStatementName)
 	paramCount := len(paramCountRegex.FindAllString(qi.queryText, -1))
+	if paramCount == 0 {
```

Comment:
Not sure I got the reason for this if else here. wouldn't it be better to just do `if paramcount > 0`?

Reply:
Uggh.. yes.. I at first thought I'd need an if/else, but then defaulted to just an if logic, but forgot to remove it. 🤦‍♂️